src/scancodes.rs

```diff
@@ -394,12 +394,13 @@ impl ScancodeSet for ScancodeSet2 {
             0x7D => Ok(KeyCode::Numpad9),            // 7D
             0x7E => Ok(KeyCode::ScrollLock),         // 7E
             0x83 => Ok(KeyCode::F7),                 // 83
+            0x84 => Ok(KeyCode::AltSysReq),          // 84
```

Comment:
Do you have a reference for this one? Microsoft have it as UNASSIGNED in their table.

Reply:
I've verified this with my Cherry PS/2 keyboard and indeed got this scancode back for Alt+SysReq.